dataprofiler/profilers/float_column_profile.py

```diff
@@ -131,21 +131,10 @@ def profile(self):
         Property for profile. Returns the profile of the column.
         :return:
         """
-
-        profile = dict(
-            min=self.np_type_to_type(self.min),
-            max=self.np_type_to_type(self.max),
-            sum=self.np_type_to_type(self.sum),
-            mean=self.np_type_to_type(self.mean),
-            variance=self.np_type_to_type(self.variance),
-            stddev=self.np_type_to_type(self.stddev),
-            skewness=self.np_type_to_type(self.skewness),
-            kurtosis=self.np_type_to_type(self.kurtosis),
+        profile = NumericStatsMixin.profile(self)
+        additive_profile = dict(
```

Comment:
i would suggest a naming change or not having an intermediary variable. Instead, just put the dict directly inside of `update`:
e.g. `profile.update({...})` 
or for var name change `float_specific_profile`.

Reply:
same for below vars

Reply:
i can see the naming intent for abstraction, but not sure the name itself makes sense to me.
The other way to do it would be to reverse the roles and do:
```
profile = {...}
profile.update(NumericStatsMixin.profile(self))
```

Reply:
I've changed it to 
- update from numericstatsmixin
- update with the specific stats using profile.udpate